configure.ac

```diff
@@ -317,6 +317,20 @@ AM_CONDITIONAL([MAINTAINER_TOOLS],[test x"${i_do_have_maintainer_tools}" = xyes]
 AC_SUBST([MAINTAINER_TOOLS],["${i_do_have_maintainer_tools}"])
 
 #--------------------------------------------------------------------------
+# Check for autotrace and potrace (don't do anything with this yet)
```

Comment:
@jtanx, this patch was a quick solution to what I saw as a problem, and it is not all that important if "said-user" saw the ./configure results or not.
To put more perspective to this...
From the last release tag until now, Fontforge has had over 100,000 downloads.
If only 1% happened to see the results of ./configure, that would be 1000 users who saw autotrace/potrace in the end-results.
If 1% of those users happen to be on the mailing list, that is 10 potential users who could also help answer the very same basic question you had to answer yourself.

We have X hours in the day, and the question(s) you don't need to answer, means more time available for hunting bugs and development instead of answering questions.
I think 1% is a bit pessimistic, therefore some numbers higher than 1% and 1% means more potential eyeballs able to help.
At the moment, I think the patch helps for now (better solution than nothing).
I agree with you documentation is a better solution, and when you write it, I'll be happy to help review/merge it.